apps/backend/src/modules/users/user.service.ts

```diff
@@ -26,12 +26,16 @@ class UserService implements Service {
 		const item = await this.userRepository.create(
 			UserEntity.initializeNew({
 				email: payload.email,
+				firstName: payload.firstName,
+				lastName: payload.lastName,
 				passwordHash: hash,
 				passwordSalt: salt,
 			}),
 		);
 
-		return item.toObject();
+		const object = item.toObject();
+
+		return { email: object.email, id: object.id };
```

Comment:
```suggestion
		return item.toObject();
```
`item` a bad name. use `user` instead